Source/WebKit/UIProcess/API/haiku/WebView.cpp

```diff
@@ -62,7 +62,7 @@ BWebView::BWebView(BRect frame,BWindow* myWindow)
 
 }
 
-void BWebView::navigationCallbacks(BLooper* app)
+void BWebView::navigationCallbacks(BWebView* app)
```

Comment:
If this points to the BWebView, the pointer should not be named "app" anymore.
And why is there a pointer to the BWebView here? Why not use the "this" pointer instead?

Reply:
Yeah forgot to change back 😅